chpl/chpl-service/src/main/java/gov/healthit/chpl/certifiedproduct/CertifiedProductDetailsManager.java

```diff
@@ -59,6 +61,12 @@ public CertifiedProductSearchDetails getCertifiedProductDetails(Long certifiedPr
         return listingService.createCertifiedSearchDetails(certifiedProductId);
     }
 
+    @Transactional(readOnly = true)
+    @Cacheable(value = CacheNames.LISTING_DETAILS)
```

Comment:
I really like the use of this cache and would like to use it in lots of other places when returning listing details to the user! Should you add some places where an entry in the cache will expire when that listing is updated? Not 100% sure how that works because we tend to just clear out entire caches vs doing it for only the one key/value pair that would have changed but I'm sure that is standard ehcache stuff. I know people probably aren't updating listings in the middle of the night when the jobs run but if it did happen I think we'd want to be using the correct data.